Source/WebCore/platform/graphics/ImageTypes.h

```diff
@@ -68,6 +68,11 @@ enum class GammaAndColorProfileOption {
     Ignored
 };
 
+enum class DecodeToHDR {
```

Comment:
Do we need both this and `DecodingFormat`?

Reply:
No. This one is removed.